tests/lib/debugRequests.test.ts

```diff
@@ -69,13 +57,13 @@ it('should warn about the code making a request that has not being mocked', asyn
 })
 
 it('should warn about the code making a request that has not being mocked enough times', async () => {
-  const consoleWarn = jest.spyOn(console, 'warn').mockImplementation()
+  const consoleWarn = vi.spyOn(console, 'warn')
   configure({ mount: render })
   wrap(MyComponentWithFeedback)
     .withNetwork({
       host: 'my-host',
       path: '/path/to/save/',
-      method: 'post',
+      method: 'POST',
```

Comment:
is it case sensitive?

Reply:
Not anymore, lowercase should work too, changing it